### DISCUSSION TITLE
XauthError with SeleniumBase UC mode on WSL

### DISCUSSION THREAD

**Discussion by subradrad** [seleniumbase/SeleniumBase#2941] [Q&A] [answered]
Environment:
Operating System: Windows Subsystem for Linux (WSL)
SeleniumBase Version: 4.28.6

```
from seleniumbase import SB

url = 'https://gitlab.com/users/sign_in'
with SB(uc=True, headed=True) as sb:
    sb.uc_open_with_reconnect(url, 15)
    sb.uc_gui_click_cf()

```
Issue:
When running a SeleniumBase script  sb.uc_gui_click_cf() , I encounter an XauthError stating that the .Xauthority file cannot be found. This issue does not occur on my standalone Windows or Linux environments.

```
Error Messages:
XauthError: FileNotFoundError: [Errno 2] No such file or directory: '/home/[username]/.Xauthority'

```
Am I missing something in my setup or configuration that could be causing this issue on WSL? 
Why does the script attempt to reinstall PyAutoGUI every time it runs?

Thank you for any assistance!

**Comment by mdmintz** [comment, marked as answer]
Looks like a case of https://superuser.com/questions/806637/xauth-not-creating-xauthority-file
Depending on your system configuration, you may encounter it. Follow the steps in the solution there to resolve.
